### DISCUSSION TITLE
Support embedded ATTACH property

### DISCUSSION THREAD

**Discussion by calebdw** [spatie/icalendar-generator#103] [Ideas]
Hello!

We would like to be able to send embedded attachments with our Calendar invites to our customers but this is not currently an option.

Per [RFC 5545](https://www.rfc-editor.org/rfc/rfc5545#section-3.8.1.1) the iCalendar format does supports embedded attachments, would y'all consider adding this functionality?

Thanks!

Edit: PR #104

**Comment by freekmurze** [comment]
Feel free to send a PR. If the code is simple, well tested and documented, there's a high change we'll merge it in.
